beater/config/config.go

```diff
@@ -77,6 +77,11 @@ type Config struct {
 	// the integration package to be installed, and for checking the
 	// Elasticsearch license level.
 	WaitReadyInterval time.Duration `config:"wait_ready_interval"`
+
+	// MaxConcurrentDecoders sets the limit on the number of concurrent batches
+	// that can be decoded at one time. This effectively limits the amount of
+	// memory consumed by the processors decodeing the incoming intake events.
+	MaxConcurrentDecoders uint `config:"max_concurrent_decoders"`
```

Comment:
We should label this as `beta`.

Reply:
Do we want to document it in `asciidoc`? My guess is no, but just checking

Reply:
No, let's not document at all for now.